packages/discovery-react-components/src/components/SearchFacets/SearchFacets.tsx

```diff
@@ -54,6 +58,10 @@ interface SearchFacetsProps {
    * Number of facet terms to show when list is collapsed
    */
   collapsedFacetsCount?: number;
+  /**
+   * Override default message displayed when receiving an error on server request
+   */
+  serverErrorMessage?: string | React.ReactElement;
```

Comment:
should we also allow `null` or `ReactNode` instead? also isn't `string` a subset of `ReactElement`?

Reply:
The props are an object, so that would mean allowing users to set `searchErrorMessage: null` -- do we need to do that? And you are right, `string` is a subset of `ReactElement`. I'll make that change.

Reply:
Went with `ReactNode`, but had to slightly change the render method. All good now, thanks.